www_src/pages/page/page.jsx

```diff
@@ -92,7 +100,11 @@ var Page = React.createClass({
         </div>
       </div>
 
-      <div className={classNames({overlay: true, active: this.state.showAddMenu})} onClick={this.toggleAddMenu}/>
+      <div className="overlaydiv">
+        <ReactCSSTransitionGroup transitionName="overlay">
+          {placementOverlay}
```

Comment:
given that we've collapsed `placementOverlay` to just this to just `<div/>` now, let's take out the bit up top and change this line to the following instead:
```
{ this.state.disableButton ? <div/> : false }
```
and then this is totally an R+